salesforce-analytics-cloud/3.2/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,397 @@
+and the public key (`salesforce-cert.crt`) files in your workspace.
+
+== Add the Connector in Studio
+
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type "analytics" in the search field.
+. Click this connector's name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+=== Configure the Connector in Studio
+
+. Drag the desired Salesforce Analytics operation to the Studio canvas.
+. To create a global element for the connector, on the *General* tab, configure the authentication as described in the authentication sections.  
++
+The following authentication connection types are available: +
++
+* <<basic-authentication,Username Password>>
+* <<oauth2,OAuth 2.0>>
+* <<oauth2-jwt,OAuth 2.0 JWT>>
+* <<oauth2-saml,OAuth 2.0 SAML Bearer>>
+. Configure these fields to upload external data into new dataset and start the processing operation:
+** Type +
+Type of the records to insert. Select a JSON file representing the schema of the dataset to be created.
+** Records +
+DataSense expression (the records to be inserted).
+** Operation +
+Specify the operation to use when you’re loading data into a dataset.
+** Description
+** Label
+** Data Set Name
++
+image::salesforce-analytics-studio-upload-and-start.png[Upload external data]
+
+[[basic-authentication]]
+=== Username Password
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Basic authentication:
+
+* *Username* +
+Enter the Salesforce username.
+* *Password* +
+Enter the corresponding password.
+* *Security Token* +
+Enter the corresponding security token.
+
+The following screenshot shows an example of configuring Basic authentication:
+
+.Basic authentication fields
+image::salesforce-analytics-basic.png[To configure authentication, select `Basic` in the *Connection* field and then complete the fields on the *General* tab.]
+
+[[oauth2]]
+=== OAuth 2.0
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure OAuth 2.0 authentication:
+
+* *Consumer Key* +
+Consumer key for the Salesforce connected app.
+* *Consumer Secret* +
+Consumer secret for the connector to access Salesforce.
+
+The following screenshot shows an example of configuring the OAuth 2.0 authentication:
+
+.OAuth 2.0 authentication fields
+image::salesforce-analytics-oauth.png[To configure authentication, select `OAuth 2.0` in the *Connection* field and then complete the fields on the *General* tab.]
+
+[[oauth2-jwt]]
+=== OAuth 2.0 JWT
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure OAuth 2.0 JWT authentication:
+
+* *Consumer Key* +
+Consumer key for the Salesforce connected app
+* *Keystore File* +
+See <<generating-keystore,Generating a Keystore File>>
+* *Store Password* +
+Password for the keystore
+* *Principal* +
+Salesforce username 
```

Comment:
```suggestion
* *Consumer Key* +
Consumer key for the Salesforce connected app.
* *Keystore File* +
See <<generating-keystore,Generating a Keystore File>>
* *Store Password* +
Password for the keystore.
* *Principal* +
Salesforce username.
```

Reply:
Same with all the similar sections.

Reply:
These aren't sentence, so no period for sentence fragments